src/components/include/cos_asm_simple_stacks.h

```diff
@@ -1,13 +1,17 @@
 #ifndef COS_ASM_SIMPLE_STACKS_H
 #define COS_ASM_SIMPLE_STACKS_H
 
+#ifndef MAX_STACK_SZ_BYTE_ORDER
+#error "Missing MAX_STACK_SZ_BYTE_ORDER, try including consts.h"
```

Comment:
I'm not one to complain about error checking, but this style of error reporting seems unconventional compared to the rest of Composite's codebase.